bartender/thrift/handler.py

```diff
@@ -81,13 +81,25 @@ def initializeInstance(self, instance_id):
         admin_args = {'auto_delete': True}
         admin_queue = self.clients['pika'].setup_queue(admin_keys[-1], admin_args, admin_keys)
 
+        connection = {
+            'host': bartender.config.publish_hostname,
+            'port': bartender.config.amq.connections.message.port,
+            'user': bartender.config.amq.connections.message.user,
+            'password': bartender.config.amq.connections.message.password,
```

Comment:
should we be saving the salted password instead?

Reply:
We can't - we need to be able to hand it to the plugins so they can use it.

Securely creating / storing passwords for different plugins is definitely a thing we should do as part of beer-garden/beer-garden#62.